components/raftstore/src/store/snap/io.rs

```diff
@@ -176,6 +201,27 @@ where
     };
 
     let instant = Instant::now();
+    let _io_type_guard = WithIoType::new(if for_balance {
+        IoType::LoadBalance
+    } else {
+        IoType::Replication
+    });
+    let mut prev_io_bytes = get_thread_io_bytes_stats().unwrap();
```

Comment:
It would be advisable to wrap the Error if `get_thread_io_bytes_stats` encounters any unexpected conditions that prevent access to `bytes_stats`.

Reply:
WithIoType::new() -> set_io_type() has already ensured that get_thread_io_bytes_total will not return an error. There’s no need to handle errors specifically here.